backend/src/main/java/ro/utcn/sd/mdantonio/StackUnderflow/controller/TagController.java

```diff
@@ -0,0 +1,64 @@
+package ro.utcn.sd.mdantonio.StackUnderflow.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.*;
+import ro.utcn.sd.mdantonio.StackUnderflow.dto.PostDTO;
+import ro.utcn.sd.mdantonio.StackUnderflow.dto.TagDTO;
+import ro.utcn.sd.mdantonio.StackUnderflow.service.PostManagementService;
+import ro.utcn.sd.mdantonio.StackUnderflow.service.TagManagementService;
+import ro.utcn.sd.mdantonio.StackUnderflow.service.UnderflowUserManagementService;
+import ro.utcn.sd.mdantonio.StackUnderflow.service.UserSessionManagementService;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class TagController {
+    private final TagManagementService tagManagementService;
+    private final PostManagementService postManagementService;
+    private final UnderflowUserManagementService underflowUserManagementService;
+    private final UserSessionManagementService userSessionManagementService;
+
+    @GetMapping("/tags")
+    public List<TagDTO> readTags() {
+        return tagManagementService.listAllTags();
+    }
+
+    @PostMapping("/tags")
+    public TagDTO createTag(@RequestBody TagDTO dto) {
+        return tagManagementService.createTag(dto.getTitle());
+    }
+
+    @DeleteMapping("/tags")
```

Comment:
Same as above: the ID should be in the URL. See https://slides.com/spet/utcn-sd-assignment-3#/3/2